tests/framework/installer/install_helper.go

```diff
@@ -347,6 +358,17 @@ func (h *InstallHelper) UninstallRookFromMultipleNS(helmInstalled bool, systemNa
 	h.k8shelper.Clientset.RbacV1beta1().ClusterRoleBindings().Delete("rook-agent", nil)
 	h.k8shelper.Clientset.RbacV1beta1().ClusterRoleBindings().Delete("anon-user-access", nil)
 	logger.Infof("done removing the operator from namespace %s", systemNamespace)
+	// removing data dir if exists
+	if len(h.dataDir) > 0 {
+		err = h.cleanupDir(h.dataDir)
```

Comment:
the cleanup pod will complete as some undeterministic time after this call, right? Are the data directories different between each test suite? If not, they could conflict

Reply:
the data directories are different (they are isolated by test names and namespaces)